### PR TITLE
Updated plugins populate_maptable to handle keyerror

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

-  **Issue :**

```
plugins = tio.plugins.list()
plugins.populate_maptable = True
for plugin in plugins:
    pprint(plugin)
```
getting below error when plugin not found in family
```
Traceback (most recent call last):
  File "REDACTED", line 9, in <module>
    for plugin in plugins:
  File "REDACTED", line 86, in __next__
    return self.next()
  File "REDACTED", line 83, in next
    fid = self._maptable['plugins'][item['id']]
KeyError: 32
```

-  **changes done :**
Assigned default value to family_id and family_name when not found

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested by running below program
```
plugins = tio.plugins.list()
plugins.populate_maptable = True
for plugin in plugins:
    pprint(plugin)
```

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by valliammal — COMMENTED** [review]
standard import "from datetime import date" should be placed before "from .base import TIOEndpoint, TIOIterator" (wrong-import-order)

**Review by valliammal — COMMENTED** [review]
Changes are approved, Very nice

**Review by valliammal — APPROVED** [review]
I approved the changes

File: tenable/io/plugins.py [hunk 1]
```diff
@@ -80,15 +80,22 @@ def next(self):
...
         # on to to the item.
         if self._maptable:
-            fid = self._maptable['plugins'][item['id']]
-            item['family_id'] = fid
-            item['family_name'] = self._maptable['families'][fid]
+            try:
+                fid = self._maptable['plugins'][item['id']]
+                item['family_id'] = fid
+                item['family_name'] = self._maptable['families'][fid]
+            except:
```
Comment (levwais) [inline tenable/io/plugins.py:87]: maybe we should log a warning with the error caught here?
use:
`self._log.warn`

Comment (SteveMcGrath) [reply tenable/io/plugins.py:87]: The issue is that non-nessus plugins are being passed back in the API.  Things such as OT and NNM plugins.

Comment (tushar-balwani) [reply tenable/io/plugins.py]: Added warning log to display ids which are not found

**Review by levwais — APPROVED** [review]

**Pull request merged**